packages/react-i18n/src/i18n.ts

```diff
@@ -318,11 +318,17 @@ export class I18n {
             ...dateStyle[style],
           });
         default:
-          return this.formatDate(date, {...formatOptions, ...dateStyle[style]});
+          return this.formatDate(date, {
```

Comment:
This seems to be just a whitespace change for the sake of it. Can we revert it to make the diff easier to review?